src/index.html

```diff
@@ -1,12 +1,444 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="page-header">
+      <a class="page-header__link-logo" href="#home">
+        <img
+        src="./images/logo.png"
+        alt="LogoMoyo"
+        class="page-header__link-images"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li data-qa="nav-hover" class="nav__item">
+            <a href="#laptops" class="nav__link">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog-container">
+      <div
+        class="card"
+        data-qa="card"
+        >
+        <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__images"
+          >
```

Comment:
```suggestion
        <img
          src="./images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
          class="card__images"
        >
```

fix your code styles